saltproc/depcode.py

```diff
@@ -159,20 +166,346 @@ def write_mat_file(self, dep_dict, dep_end_time):
         """
 
 
+class DepcodeOpenMC(Depcode):
+    r"""Class contains information about input, output, geometry, and
+    template files for running OpenMC depletion simulations.
+    Also contains neutrons population, active, and inactive cycles.
+    Contains methods to read template and output files,
+    write new input files for OpenMC.
+
+    Attributes:
+    -----------
+    param : dict of str to type
```

Comment:
Again, not clear. What str and what type? Could provide more info in the next line

Reply:
I expanded the docstring.